app/views/comments/_comment.html.erb

```diff
@@ -93,6 +93,7 @@
         </div>
       <% end %>
     </li>
+    <%unless child_comments_of(comment).empty?%>
```

Comment:
@iagirre to be consistent with the rest of the erb code, spaces before `<%` and after `%>` would be nice 👍